src/api/schemas/user-schema.ts

```diff
@@ -92,5 +94,17 @@ const updateUserSchema = object({
     })
 });
 
+const addNewUserActivity = object({
+    params: object({
+        id: firebase_id
+    }).strict('Request URL contains an invalid key'),
+    body: object({
+        // Activity
+        activity: activitySchema.strict('Activity field contains an invalid key'),
```

Comment:
The activity schema should also contain images (not urls) and also an  'address' field